save-cosv-frontend/README.md

```diff
@@ -0,0 +1,168 @@
+# COSV Frontend
```

Comment:
does this file duplicates the readme from `save-frontend`?

Reply:
Yes. The frontend for COSV contains the same instructions as the Save frontend. The only difference is in the launch command.

Reply:
may be leave one readme in `save-frontend-common`, and link readmes from `save-frontend` and `save-cosv-frontend` to this common readme? like see 'save-frontend-common/readme'?

Reply:
Done. I will delete duplicate information from Readme in the `save-frontend` in the next PR.